Debian's 00upstream_fix_annotations.patch: Fix wrong type names in annotations.
diff --git a/libsocialweb-client/sw-client-contact-view.c b/libsocialweb-client/sw-client-contact-view.c
index bb2679f..6eaee0e 100644
--- a/libsocialweb-client/sw-client-contact-view.c
+++ b/libsocialweb-client/sw-client-contact-view.c
@@ -384,7 +384,7 @@ sw_client_contact_view_class_init (SwClientContactViewClass *klass)
   /**
    * SwClientContactView::contacts-added:
    * @self:
-   * @contacts: (type GLib.List) (element-type Sw.Contact):
+   * @contacts: (type GLib.List) (element-type SocialWebClient.Contact):
    */
   signals[CONTACTS_ADDED_SIGNAL] =
     g_signal_new ("contacts-added",
@@ -401,7 +401,7 @@ sw_client_contact_view_class_init (SwClientContactViewClass *klass)
   /**
    * SwClientContactView::contacts-removed:
    * @self:
-   * @contacts: (type GLib.List) (element-type Sw.Contact):
+   * @contacts: (type GLib.List) (element-type SocialWebClient.Contact):
    */
   signals[CONTACTS_REMOVED_SIGNAL] =
     g_signal_new ("contacts-removed",
@@ -418,7 +418,7 @@ sw_client_contact_view_class_init (SwClientContactViewClass *klass)
   /**
    * SwClientContactView::contacts-changed:
    * @self:
-   * @contacts: (type GLib.List) (element-type Sw.Contact):
+   * @contacts: (type GLib.List) (element-type SocialWebClient.Contact):
    */
   signals[CONTACTS_CHANGED_SIGNAL] =
     g_signal_new ("contacts-changed",
diff --git a/libsocialweb-client/sw-client-item-view.c b/libsocialweb-client/sw-client-item-view.c
index 6bfdb76..ab98fdd 100644
--- a/libsocialweb-client/sw-client-item-view.c
+++ b/libsocialweb-client/sw-client-item-view.c
@@ -389,7 +389,7 @@ sw_client_item_view_class_init (SwClientItemViewClass *klass)
   /**
    * SwClientItemView::items-added:
    * @self:
-   * @items: (type GLib.List) (element-type Sw.Item):
+   * @items: (type GLib.List) (element-type SocialWebClient.Item):
    */
   signals[ITEMS_ADDED_SIGNAL] =
     g_signal_new ("items-added",
@@ -406,7 +406,7 @@ sw_client_item_view_class_init (SwClientItemViewClass *klass)
   /**
    * SwClientItemView::items-removed:
    * @self:
-   * @items: (type GLib.List) (element-type Sw.Item):
+   * @items: (type GLib.List) (element-type SocialWebClient.Item):
    */
   signals[ITEMS_REMOVED_SIGNAL] =
     g_signal_new ("items-removed",
@@ -423,7 +423,7 @@ sw_client_item_view_class_init (SwClientItemViewClass *klass)
   /**
    * SwClientItemView::items-changed:
    * @self:
-   * @items: (type GLib.List) (element-type Sw.Item):
+   * @items: (type GLib.List) (element-type SocialWebClient.Item):
    */
   signals[ITEMS_CHANGED_SIGNAL] =
     g_signal_new ("items-changed",
